kolibri/core/assets/src/views/__tests__/FocusTrap.spec.js

```diff
@@ -0,0 +1,60 @@
+import { render } from '@testing-library/vue';
+import userEvent from '@testing-library/user-event';
+import VueRouter from 'vue-router';
+import FocusTrap from '../FocusTrap.vue';
+
+const renderComponent = (props = {}) => {
```

Comment:
The FocusTrap tests look good.

Not a blocker -- the only thing I could think that might be worth adding is a call to the public `reset` method on FocusTrap to see that it updates `isTrapActive` and such.

Reply:
Got it! That's an important use case. Will add tests for it.

Reply:
I have added a test suite to cover this use-case, but I can't get it to work. Like to access the public method `reset` of the component, we would need to add a wrapper around it so that the user can access the same. I tried writing a test suite with the same, but can't get it to work. Would highly appreciate any help with the same.

Reply:
I think this is a case where the 'user' of the component is actually the developer, so I don't think it's necessary to do this via DOM interaction, and the reset method can be invoked programmatically - as it forms part of the public API of the component.